spec/requests/api/reservations_spec.rb

```diff
@@ -0,0 +1,96 @@
+require 'swagger_helper'
+# include JsonWebToken
```

Comment:
- [ ] Kindly delete these pieces of old commented code in order to improve the cleanliness of the repo.  💪